data/SkyrimSE/po3/wiki.ts

```
@@ -0,0 +1,399 @@
+                        }
+
+                        case 'code':
+                            continue; // these code blocks are just the Papyrus declaration, which we already have
+
+                        default:
+                            if (functionUnderConstruction.node) functionUnderConstruction.description.push(node);
+                    }
+                }
+                finalizeFunction(file.ast.children.at(-1) ?? file.ast);
+
+                if (i === file.ast.children.length) continue fileLoop;
```

Biome (1.9.4)

[error] Unnecessary label.

Safe fix: Remove the unnecessary label.
You can achieve the same result without the label.


(lint/complexity/noUselessLabel)